### PR TITLE
Clean and improve something.

### PR DESCRIPTION
**Pull request by erkam2002**

### CODE REVIEW

**Comment by DerddyDert** [comment]
@erkam2002 you test it?

**Comment by erkam2002** [comment]
What test?

**Comment by DerddyDert** [comment]
Your src?

**Comment by MasterOfTheRealm** [comment]
Please tell us if you test your commits @erkam2002

**Comment by erkam2002** [comment]
I dont want to test it but i know it will work

**Comment by MasterOfTheRealm** [comment]
you never know...please test it

**Comment by SuperXingKong** [comment]
For what?

**Comment by DerddyDert** [comment]
@erkam2002 did you test yet??

**Comment by erkam2002** [comment]
You mean test that the server starts without error? or What?

**Comment by erkam2002** [comment]
Or test setMovementSpeed?

**Comment by xISRAPILx** [comment]
all work, i'm tested! @erkam2002 @DerddyDert

**Comment by erkam2002** [comment]
speed

**Comment by erkam2002** [comment]
I tested it too -.- xD

**Comment by DerddyDert** [comment]
Cool vid @erkam2002

**Comment by SuperXingKong** [comment]
I think setMovement is great, but there is no point in changing others. @DerddyDert @Leo3418

**Comment by erkam2002** [comment]
?

File: src/pocketmine/Player.php [hunk 1]
```diff
@@ -4049,6 +4036,8 @@ public function sendTitle($title, $subtitle = "", $fadein = 20, $fadeout = 20, $
 		$this->chunk = null;
 
 		$this->server->removePlayer($this);
+
+        parent::close();
```
Comment (SuperXingKong) [inline src/pocketmine/Player.php]: Why did you move it here?

Comment (erkam2002) [inline src/pocketmine/Player.php]: looks good xD

Comment (SuperXingKong) [inline src/pocketmine/Player.php]: It caused that chest disappear...

Comment (SuperXingKong) [inline src/pocketmine/Player.php]: I will revert it, testing.

Comment (erkam2002) [inline src/pocketmine/Player.php]: that is sendTitle...

**Comment by DerddyDert** [comment]
I like when @SuperXingKong comes and fixes everything :-p this one is my fault sorry

**Comment by SuperXingKong** [comment]
@DerddyDert Thanks xD
